.env

```diff
@@ -1,12 +1,11 @@
-REACT_APP_PORTIS_ID=78552035-b1c5-4927-8d2f-4eeea42a795c
-REACT_APP_FORTMATIC_KEY=pk_live_EF523DF5F8AC3058
-REACT_APP_MAINNET_ENV=production
+REACT_APP_MAINNET_ENV=staging
 
-REACT_APP_AGGREGATOR_API=https://aggregator-api.kyberswap.com
+REACT_APP_META_AGGREGATOR_API=https://aggregator-api.kyberswap.com
```

Comment:
I dont see usage of this env?

Reply:
We do use it. I have refactored these env into env.ts file. Please search for usage of `META_AGGREGATOR_API`, it will shown 37 result in 15 files.

Reply:
I mean we only need 1 endpoint.

Reply:
Confirmed with matthew, will be same domain, remove META env please

Reply:
done